deebot_client/commands/json/map.py

```diff
@@ -22,6 +24,14 @@ class GetCachedMapInfo(JsonCommandWithMessageHandling, MessageBodyDataDict):
     """Get cached map info command."""
 
     name = "getCachedMapInfo"
+    # version definition for using type of getMapSet v1 or v2
+    get_map_set_version: int = 1
```

Comment:
```suggestion
```

We shouldn't create a class variable for it

Reply:
updated to use the match-case option